lib/saml_idp/assertion_builder.rb

```diff
@@ -56,58 +48,61 @@ def initialize(
-            conditions.AudienceRestriction do |restriction|
-              restriction.Audience audience_uri
-            end
+        end
+        assertion.Conditions NotBefore: not_before,
+                             NotOnOrAfter: not_on_or_after_condition do |conditions|
+          conditions.AudienceRestriction do |restriction|
+            restriction.Audience audience_uri
           end
-          if asserted_attributes
-            assertion.AttributeStatement do |attr_statement|
-              asserted_attributes.each do |friendly_name, attrs|
-                attrs = (attrs || {}).with_indifferent_access
-                attr_statement.Attribute Name: attrs[:name] || friendly_name,
-                  NameFormat: attrs[:name_format] || Saml::XML::Namespaces::Formats::Attr::URI,
-                  FriendlyName: friendly_name.to_s do |attr|
-                    values = get_values_for friendly_name, attrs[:getter]
-                    values.each do |val|
-                      attr.AttributeValue val.to_s
-                    end
-                  end
+        end
+        if asserted_attributes
+          assertion.AttributeStatement do |attr_statement|
+            asserted_attributes.each do |friendly_name, attrs|
+              attrs = (attrs || {}).with_indifferent_access
+              attr_statement.Attribute Name: attrs[:name] || friendly_name,
+                                       NameFormat: attrs[:name_format] || Saml::XML::Namespaces::Formats::Attr::URI,
+                                       FriendlyName: friendly_name.to_s do |attr|
+                values = get_values_for friendly_name, attrs[:getter]
+                values.each do |val|
+                  attr.AttributeValue val.to_s
+                end
               end
             end
           end
-          assertion.AuthnStatement AuthnInstant: now_iso, SessionIndex: reference_string do |statement|
-            statement.AuthnContext do |context|
-              case authn_context_classref
-              when Array
-                context.AuthnContextClassRef(
-                  authn_context_classref.select { |classref| %r{/ial/}.match? classref }.first
-                )
-              else
-                context.AuthnContextClassRef authn_context_classref
-              end
+        end
+        assertion.AuthnStatement AuthnInstant: now_iso,
+                                 SessionIndex: reference_string do |statement|
+          statement.AuthnContext do |context|
+            case authn_context_classref
+            when Array
+              context.AuthnContextClassRef(
+                authn_context_classref.select { |classref| %r{/ial/}.match? classref }.first
+              )
+            else
+              context.AuthnContextClassRef authn_context_classref
             end
           end
         end
+      end
     end
-    alias_method :raw, :fresh
+    alias raw fresh
```

Comment:
I'm surprised it converted `alias_method` to plain `alias`! I thought `alias_method` was preferred because it can be overridden the normal way

Reply:
fixed in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a, and i added a cop to keep it from happening again